front/app/containers/Admin/projects/project/nativeSurvey/FormResults/FormResultsPage/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import {
+  Box,
+  colors,
+  Text,
+  Title,
+  Tooltip,
+} from '@citizenlab/cl2-component-library';
+import { useTheme } from 'styled-components';
+
+import { LogicConfig, ResultUngrouped } from 'api/survey_results/types';
+
+import useLocalize from 'hooks/useLocalize';
+
+import LogicIcon from 'containers/Admin/projects/project/nativeSurvey/FormResults/LogicIcon';
+
+import { useIntl } from 'utils/cl-intl';
+
+import messages from '../messages';
+
+type FormResultsPageProps = {
+  result: ResultUngrouped;
+  totalSubmissions: number;
+  logicConfig: LogicConfig;
+};
+
+const FormResultsPage = ({
+  result,
+  totalSubmissions,
+  logicConfig,
+}: FormResultsPageProps) => {
+  const localize = useLocalize();
+  const { formatMessage } = useIntl();
+  const theme = useTheme();
+
+  const {
+    question,
+    pageNumber,
+    totalResponseCount,
+    questionResponseCount,
+    hidden,
+    customFieldId,
+    logic,
+  } = result;
+
+  const questionTitle = localize(question);
+  const pageTitle = `${formatMessage(messages.page)} ${pageNumber}${
+    questionTitle ? `: ${questionTitle}` : ''
+  }`;
+
+  if (hidden) {
+    return (
+      <Box borderLeft={`4px solid ${colors.coolGrey300}`} pl="12px">
+        <Text color={'grey500'}>
+          {pageTitle} {formatMessage(messages.hiddenByLogic)}
+        </Text>
+      </Box>
+    );
+  }
+
+  return (
+    <Box
+      data-cy={`e2e-survey-result-page-${pageNumber}`}
+      mb="24px"
+      pl="12px"
+      borderLeft={`4px solid ${theme.colors.tenantPrimary}`}
+    >
+      <Title variant="h3" mt="12px" mb="12px">
+        {pageTitle}
+      </Title>
+      <Tooltip
+        disabled={totalResponseCount === totalSubmissions}
+        placement="bottom-start"
+        content={formatMessage(messages.resultsCountQuestionTooltip)}
+        theme="dark"
+      >
+        <Box display="flex">
+          <Text variant="bodyS" color="textSecondary" mt="0" mb="4px">
+            {`${questionResponseCount}/${totalResponseCount} ${formatMessage(
```

Comment:
Better would be:

```
formatMessage(messages. responses, { questionResponses: questionResponseCount, questionSeenBy: totalResponseCount })
```

And then in the message:

```
defaultMessage: "{questionResponses}/{questionSeenBy} responses"
```

in case this type of format is different in other languages